packages/hardhat-toolbox/test/fixture-projects/only-toolbox/script.js

```diff
@@ -9,6 +9,9 @@ async function main() {
   assert(taskNames.includes("verify"));
   assert(taskNames.includes("coverage"));
   assert(taskNames.includes("typechain"));
+  // TODO: Ignition isn't appearing the task list,
+  // though it is imported as part of the toolbox
+  // assert(taskNames.includes("ignition"));
```

Comment:
I attempted to include a check for Ignition (and would do the same for the viem toolbox) but the tasks don't appear in the sample project.
I am unsure if this is because the toolbox version has not been updated yet.

Reply:
@zoeyTM spotted that the issue is that `ignition` is a scoped task.

We have added a check for appropriate scopes:
```
  // check that the expected scopes are there
  const scopeNames = Object.keys(scopes);
  assert(scopeNames.includes("vars"));
  assert(scopeNames.includes("ignition"));
```